engine/src/cmd/base_interface.cpp

```diff
@@ -1273,8 +1271,7 @@ extern void PlayDockingSound(int dock);
 void BaseInterface::Room::Launch::Click(BaseInterface *base, float x, float y, int button, int state) {
     if (state == WS_MOUSE_UP) {
         Link::Click(base, x, y, button, state);
-        static bool
-                auto_undock_var = XMLSupport::parse_bool(vs_config->getVariable("physics", "AutomaticUnDock", "true"));
+        const bool auto_undock_var = configuration()->physics.automatic_undock;
```

Comment:
Why do we have this? Why not just reference configuration()->physics.automatic_undock directly from _if ((playa && bas) && (auto_undock || ..._

Reply:
Done.